graph/memstore/quadstore.go

```diff
@@ -193,7 +193,7 @@ func (qs *QuadStore) indexOf(t quad.Quad) (int64, bool) {
 
 func (qs *QuadStore) AddDelta(d graph.Delta) error {
 	if _, exists := qs.indexOf(d.Quad); exists {
-		return graph.ErrQuadExists
+		return &graph.DeltaError{Delta: d, Err: graph.ErrQuadExists}
```

Comment:
AddDelta and RemoveDelta already have the error wrapped. The invalid action does though.